ArraysHomework/ArraysHomework.cs

```diff
@@ -139,14 +182,20 @@ public void Lists()
             // just a little fun to fill them
             int up = 0;
             int down = 24;
+
             for (; up < 25 && down >= 0; up++, down--)
             {
                 copy1.Add(up);
                 copy2.Add(down);
             }
-
+   
             // Your code here: combine copy1 into copy 2
 
+            for (int = i 25; int < 50; addUp++, int++)
```

Comment:
This statement doesn't compile.
Keep in mind what is a type (int), what is a variable name (i), and where the assignment operator goes.